src/main/kotlin/org/openmbee/flexo/mms/GuardedPatch.kt

```diff
@@ -156,7 +157,9 @@ suspend fun <TResponseContext: LdpMutateResponse> LdpDcLayer1Context<TResponseCo
 
 
     executeSparqlUpdate(updateString) {
-        prefixes(prefixes)
+        val p = prefixes
+        p.map.putAll(clientPrefixes)
```

Comment:
this allows client to override internal prefix mappings

Reply:
the flexo mappings are taken out at line 60